tap/line.py

```diff
@@ -116,7 +116,7 @@ class Diagnostic(Line):
     """A diagnostic line (i.e. anything starting with a hash)."""
 
     def __init__(self, text):
-        self._text = text
+        self._text = text.lstrip('# ')
```

Comment:
Is the `#` technically part of the spec as well for diagnostic lines?

Reply:
Hmm... now that you mention it, on a re-read, I'd say yes, it's probably part of the diagnostic. I do appreciate the effort.

I guess we're both reminded that specs are weird things full of weird rules.

At the very least, you did find a valuable Travis update. Maybe we can close this PR and open another one with that `.travis.yml` change.